nautobot_chatops/tests/test_api.py

```diff
@@ -1,10 +1,13 @@
-"""Test cases for Nautobot Chatops API."""
-from importlib import metadata
+"""Unit tests for nautobot_chatops."""
 
+from django.contrib.auth import get_user_model
+from django.test import TestCase
 from django.urls import reverse
+from nautobot.users.models import Token
+from rest_framework import status
+from rest_framework.test import APIClient
 
-from nautobot.core.testing import APITestCase, APIViewTestCases
-from nautobot_chatops.models import AccessGrant, CommandToken
```

Comment:
@gsnider2195 Do you know why these lines are being removed?

Reply:
Bad merge from drift manager. I had the same issue with the bgp models app